i2c/sw/sensirion_sw_i2c_gpio.h

```diff
@@ -64,7 +64,7 @@ void sensirion_SCL_out(void);
 
 uint8_t sensirion_SCL_read(void);
 
-void sensirion_sleep_usec(uint32_t useconds);
+void sensirion_i2c_hal_sleep_usec(uint32_t useconds);
```

Comment:
Isn't this function already declared in the `sensirion_i2c_hal.h`?

Reply:
Resolved offline -> It is needed to have all functions which a user may need to implement defined in a single header file. If this is really a smart idea is a different question. We'll rethink this approach in a future PR.